content/events/pl-founders-retreat.md

```diff
@@ -0,0 +1,21 @@
+---
+eventName: "PL Founders Retreat"
+eventDescription: "The PL Founders Retreat is a chance for PL network founders and leaders to recharge, build connections, and learn from each other in an informal, relaxed setting. Please note that this retreat is by invitation only." 
+
+eventTopic: 
+eventType: "Retreat" 
```

Comment:
@laeccles The eventType can have one among the following values: Conference, Virtual, Social. Please select one from the above.

Reply:
@WinsCodes I have updated this